ch2/loopingATriangle.js

```diff
@@ -0,0 +1,6 @@
+let pattern = '#';
+
+for(let i = 0; i < 7; i++) {
+	console.log(pattern);
+	pattern += '#';
```

Comment:
indentation needs to be fixed here, see https://github.com/clkwrkunvrs/c2bc/pull/1#discussion_r1534954950